### PR TITLE
Cert: Port the ItElasticLogging  usecases in OKD

### PR DESCRIPTION
**Pull request by hzhao-github**
Result from kind new:

https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/7513
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/7602

Result from okd-test

https://build.weblogick8s.org:8443/job/wko-okd-test/98/

the test case testWlsLoggingExporter is disabled in OKD env because we got permission issue when creating the domain with WLS logging exporter installment is required, WLS loging exporter jars(
weblogic-logging-exporter.jar  and snakeyaml.jar ) are downloaded successfully though. 

from ns-xpntsx.list.events.log of domain, it says:

  message: 'Failed to complete processing domain resource elk-domain1 due to: Cannot
    write a file to directory ''/u01/domains/elk-domain1'' using command ''touch /u01/domains/elk-domain1/testaccess.tmp'',
    error=''touch: cannot touch ''/u01/domains/elk-domain1/testaccess.tmp'': Permission
    denied''. DomainSourceType is ''FromModel'' and this is the DOMAIN_HOME directory
    specified by ''domain.spec.domainHome''., the processing will be retried if needed'

### CODE REVIEW

**Review by anpanigr — COMMENTED** [review]
Provide kin-new and OKD test result on the description.

**Comment by maggiehe00** [comment]
Based on the change on OKD wlsLoggingExporter installation and corresponding test method testWlsLoggingExporter are disabled.
Please add a brief explanation in the code to explain the reason as we did in other classes. If you think you might enable them in the future, please file a JIRA to track the error you are getting now.
Thanks.

File: integration-tests/pom.xml [hunk 1]
```diff
@@ -381,7 +382,7 @@
                   **/ItPodTemplates,
                   **/ItRemoteConsole,
                   **/ItServerStartPolicy,
-		  **/ItServiceStartPolicyDynamicCluster,
+		          **/ItServiceStartPolicyDynamicCluster,
```
Comment (maggiehe00) [inline integration-tests/pom.xml:385]: No extra space needed here

**Review by maggiehe00 — CHANGES_REQUESTED** [review]
Based on the change on OKD wlsLoggingExporter installation and corresponding test method testWlsLoggingExporter are disabled.
Please add a brief explanation in the code to explain the reason as we did in other classes. If you think you might enable them in the future, please file a JIRA to track the error you are getting now.
Thanks.

File: integration-tests/pom.xml (hunk 1 above)
Comment (hzhao-github) [reply integration-tests/pom.xml:385]: this test suite is existing one, not mine from this change. I made change when I saw it's indent incorrectly, I used tab to indent that works always to me. Let me try to fix it again

**Review by maggiehe00 — COMMENTED** [review]
I saw error msg: "Cannot
write a file to directory ''/u01/domains/elk-domain1'' using command ''touch /u01/domains/elk-domain1/testaccess.tmp''

Is it possible to put testaccess.tmp directly under dir /u01, instead of /u01/domains/elk-domain1. Previously we had the similar dir permission denied issue and resolution is to move up to /u01

**Review by anpanigr — COMMENTED** [review]
Add the result from OKD run in PR description

**Comment by hzhao-github** [comment]
That's great that you encountered the same issue before. However I am not sure who generated that file, seems it's created by the Operator/introspecter? how about we separate this issue with change. I created a JIRA OWLS-94986, will come back to it after I complete some more higher priority jira. Thanks

**Review by anpanigr — APPROVED** [review]
lgtm

**Review by maggiehe00 — APPROVED** [review]
LGTM

**Pull request merged**